statistics/table.go

```diff
@@ -44,8 +45,10 @@ type Table struct {
 	TableID     int64
 	Columns     map[int64]*Column
 	Indices     map[int64]*Index
-	Count       int64 // Total row count in a table.
-	ModifyCount int64 // Total modify count in a table.
+	colName2Idx map[string]int64 // map column name to index id
```

Comment:
If we maintain histogram in plan, we will need to maintain this two map too?

Reply:
Yes, or we need to brute force over all the indices and columns...